pkg/domain/infra/abi/artifact.go

```diff
@@ -132,6 +134,7 @@ func (ir *ImageEngine) ArtifactRm(ctx context.Context, opts entities.ArtifactRem
 			}
 			return nil, err
 		}
+		ir.Libpod.NewArtifactEvent(events.Remove, "", namesOrDigest, nil)
```

Comment:
These should probably go in libartifact in containers/container-libs - pass it in when creating a new instance of an artifact store (at https://github.com/containers/container-libs/blob/main/common/pkg/libartifact/store/store.go#L50) and go from there